src/main/java/Duke.java

```diff
@@ -1,29 +1,28 @@
 import java.util.Scanner;
 
 public class Duke {
-
+    static final String LOGO = "   _____                .__   \n" +
+            "  /  _  \\ ___  ___ ____ |  |  \n" +
+            " /  /_\\  \\\\  \\/  // __ \\|  |  \n" +
+            "/    |    \\>    <\\  ___/|  |__\n" +
+            "\\____|__  /__/\\_ \\\\___  >____/\n";
     public static void greetUser(){
-        String greeting = "____________________________________________________________" + System.lineSeparator()
-        + "Hello! My name is Axel. :-)" + System.lineSeparator()
-        + "How may I help you today?" + System.lineSeparator()
-        + "____________________________________________________________" + System.lineSeparator();
+        String greeting = "____________________________________________________________\n"
+        + "Hello! My name is Axel. :-)\n"
+        + "How may I help you today?\n"
+        + "____________________________________________________________";
         System.out.println(greeting);
     }
 
     public static void sayByeToUser(){
-        String message = "____________________________________________________________" + System.lineSeparator()
-                + "Goodbye. Hope to see you again soon!" + System.lineSeparator()
-                + "____________________________________________________________";
+        String message = "____________________________________________________________\n"
+        + "Goodbye. Hope to see you again soon!\n"
+        + "____________________________________________________________";
         System.out.println(message);
     }
 
     public static void main(String[] args) {
-        String logo = "   _____                .__   \n" +
-                "  /  _  \\ ___  ___ ____ |  |  \n" +
-                " /  /_\\  \\\\  \\/  // __ \\|  |  \n" +
-                "/    |    \\>    <\\  ___/|  |__\n" +
-                "\\____|__  /__/\\_ \\\\___  >____/\n";
-        System.out.println(logo);
+        System.out.println(LOGO);
```

Comment:
Good use of  the constant string and correct syntax of using capital letters for naming the constant.